docs/documentation/upgrading/topics/migrate_db.adoc

```diff
@@ -7,8 +7,8 @@ database is automatically migrated when you start the new installation for the f
 
 === Automatic relational database migration
 
-To enable automatic upgrading of the database schema, set the migration-strategy property value to "update" for the
-default connections-jpa provider:
+To enable automatic upgrading of the database schema, set the `migration-strategy` property value to "update" for the
+default `connections-jpa` provider:
```

Comment:
I think `migration-strategy=update` is the default, no?

Reply:
Yep, that's right. Could also be clarified. Nice catch!